### PR TITLE
Add few blogs and courses resource

### PR DESCRIPTION
**Pull request by hashemirafsan**

### CODE REVIEW

File: src/data/courses.json [hunk 1]
```diff
@@ -26,5 +26,9 @@
     {
         "title": "Laracasts",
         "link": "https://laracasts.com/"
+    },
+    {
+        "title": "LaraShout",
```
Comment (me-shaon) [inline src/data/courses.json]: I think it's better to add 'Larashout' to blogs, rather than in courses.

**Review by me-shaon — CHANGES_REQUESTED** [review]

File: src/data/blogs.json [hunk 2]
```diff
@@ -14,5 +14,17 @@
...
+    {
+        "title": "Laravel Daily",
+        "link": "https://laraveldaily.com"
+    },
+    {
+        "title": "Laravel Tricks",
+        "link": "https://laravel-tricks.com/tricks"
+    },
+    {
+        "title": "Laraship",
```
Comment (me-shaon) [inline src/data/blogs.json]: Seems like most of the resource in `laraship` is about their product, not about Laravel. so, it'd be better not to add it in our resource list.

**Review by me-shaon — CHANGES_REQUESTED** [review]

File: src/data/courses.json (hunk 1 above)
Comment (hashemirafsan) [reply src/data/courses.json]: Yeah, i move it to blogs from courses

File: src/data/blogs.json (hunk 2 above)
Comment (hashemirafsan) [reply src/data/blogs.json]: Removed laraship from blogs

**Comment by me-shaon** [comment]
@hashemirafsan After the recent changes, it's showing that there are no additional changes to merge. See `fils changed` section.
Screenshot 2021-08-23 at 11 36 24 AM

**Comment by hashemirafsan** [comment]
@me-shaon I think already added those resource before my pull request, we can close this PR, if there is no changes.

**Comment by me-shaon** [comment]
@hashemirafsan yeah, sorry, I've already added those locally earlier before I saw your pull request. Thanks for contributing. Looking forward to getting more contribution from you in future. :)

**Pull request closed without merging**